### DISCUSSION TITLE
UI button to lock pan/zoom

### DISCUSSION THREAD

**Discussion by AeliusSaionji** [deckerst/aves#1182] [Ideas]
I often need to zoom in on a picture to see if small details are in focus. If I take multiple similar shots, it would be convenient to be able to swipe between them to compare a zoomed-in section of the image.

This would be particularly useful for burst shots, too.

I would imagine we'd want the lock button to reset every time we leave Aves, so normal function is always default.

**Comment by deckerst** [comment]
True, I have the same need. Don't know yet how to present the UI/UX for it though.

**Reply by AeliusSaionji** [reply]
My first instinct of where to look for this would be in the viewer "quick actions".
